docs/quick-start.md

```diff
@@ -140,12 +140,12 @@ You can also use this convenient wrapper that provides the ability to change the
 
 ```bash
 function ya() {
-	tmp="$(mktemp -t "yazi-cwd.XXXXX")"
-	yazi --cwd-file="$tmp"
-	if cwd="$(cat -- "$tmp")" && [ -n "$cwd" ] && [ "$cwd" != "$PWD" ]; then
-		cd -- "$cwd"
-	fi
-	rm -f -- "$tmp"
+    tmp="$(mktemp -t "yazi-cwd.XXXXX")"
```

Comment:
It's better to keep tabs as they have better accessibility than spaces

- https://github.com/prettier/prettier/issues/7475
- https://www.reddit.com/r/javascript/comments/c8drjo/nobody_talks_about_the_real_reason_to_use_tabs/

Reply:
That's what I thought but the tabs were way too large. Did I miss some CSS setting the `tab-size`?